frontend/src/components/activePage/TableInstance/RowInstanceActions/RowInstanceActionsPersistent.tsx

```diff
@@ -49,7 +49,8 @@ const RowInstanceActionsPersistent: FC<IRowInstanceActionsPersistentProps> = ({
     }
   };
 
-  return status === 'VmiReady' ? (
+  return status === 'VmiReady' ||
```

Comment:
Should convert to the appropriate enum, since it is now defined in the generated-types.